qemu/tests/cfg/guest_iommu_group.cfg

```diff
@@ -0,0 +1,41 @@
+# Please add "iommu=pt intel_iommu=on" to host kernel line
+- guest_iommu_group: 
+    virt_test_type = qemu
+    type = guest_iommu_group
+    only virtio_blk virtio_net virtio_scsi
+    only q35
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    no Host_RHEL.m7
+    machine_type_extra_params = "kernel-irqchip=split"
+    virtio_dev_iommu_platform = on
+    nics = "nic0"
+    nic_model_nic0 = virtio
+    device_id_nic0 = nic0
+    images += " stg0 stg1"
+    image_name_stg0 = "images/stg0"
+    image_name_stg1 = "images/stg1"
+    image_size_stg0 = 1G
+    image_size_stg1 = 1G
+    force_create_image_stg0 = yes
+    force_create_image_stg1 = yes
+    remove_image_stg0 = yes
+    remove_image_stg1 = yes
+    blk_extra_params_stg0 = "serial=TARGET_DISK0"
+    blk_extra_params_stg1 = "serial=TARGET_DISK1"
+    drive_format_stg0 = virtio
+    drive_format_stg1 = scsi-hd
+    variants:
+        - iommu:
```

Comment:
So this variant only for x86, could you help to add an arm-only variant?
```
- smmu:
    only aarch64
    machine_type_extra_params += ",iommu=smmuv3"
```

Reply:
updated